srv/components/misc_pkgs/swupgrade/repo/install.sls

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+{% macro repo_added(release, source, source_type, repo_name, repo_params={}) %}
```

Comment:
macros are cryptic for unfamiliar person reading the code.
Please add some comment explaining how this is to be interpreted. It would help with community exposure.

Reply:
let's move to a separate later task as it's a copy-paste of old mount approach. @dmitrydb could you please create a separate task to cover non-yum repositories inside ISO and put there this comment as a requirement since we would anyway change the states to support non-yum repos

Reply:
Task for that:
https://jts.seagate.com/browse/EOS-16933